src/FormMigration/Steps/RazorpayPerFormSettings.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Give\FormMigration\Steps;
+
+use Give\FormMigration\Contracts\FormMigrationStep;
+
+/**
+ * @unreleased
+ */
+class RazorpayPerFormSettings extends FormMigrationStep
```

Comment:
This should have a `canHandle()` method so the attributes are only added when the v2 form is customized.  We should be checking if the meta exists before processing

Reply:
Resolved in 251a5e452a0de49f45e1fa0a67e1a860fb20be91